gpu-operator/release-notes.rst

```diff
@@ -34,6 +34,94 @@ See the :ref:`GPU Operator Component Matrix` for a list of components included i
 
 ----
 
+.. _v24.6.0:
+
+24.6.0
+======
+
+.. _v24.6.0-new-features:
+
+New Features
+------------
+
+* Added support for the NVIDIA Data Center GPU Driver version 550.90.07.
+  Refer to the :ref:`GPU Operator Component Matrix`
+  on the platform support page.
+
+* Added support for the following software component versions:
+
+    - NVIDIA Container Toolkit v1.16.1
+    - NVIDIA Driver Manager for Kubernetes v0.6.10
+    - NVIDIA Kubernetes Device Plugin v0.16.1
+    - NVIDIA DCGM Exporter v3.3.7-3.5.0
+    - NVIDIA DCGM v3.3.7-1
+    - Node Feature Discovery v0.16.3
+    - NVIDIA GPU Feature Discovery for Kubernetes v0.16.1
+    - NVIDIA MIG Manager for Kubernetes v0.8.0
+    - NVIDIA KubeVirt GPU Device Plugin v1.2.9
+    - NVIDIA vGPU Device Manager v0.2.7
+    - NVIDIA GDS Driver v2.17.5
+    - NVIDIA Kata Manager for Kubernetes v0.2.1
+    - NVIDIA GDRCopy Driver v2.4.1-1
+
+* Added support for NVIDIA Network Operator v24.4.0.
+  Refer to :ref:`Support for GPUDirect RDMA` and :ref:`Support for GPUDirect Storage`.
+
+* Added support for using precompiled driver containers that use the 550 branch Open Kernel module drivers.
+  Specify ``--set driver.useOpenKernelModules=true --set driver.usePrecompiled=true --set driver.version=550``
+  when you install or upgrade the Operator.
+  Support remains limited to Ubuntu 22.04.
+  Refer to :doc:`precompiled-drivers` for more information.
+
+* Added support for new MIG profiles.
+
+  * For H200 devices:
+
+    * ``1g.18gb``
+    * ``1g.18gb+me``
+    * ``1g.35gb``
+    * ``2g.35gb``
+    * ``3g.71gb``
+    * ``4g.71gb``
+    * ``7g.141gb``
+
+  * For H20 devices:
+
+    * ``1g.12gb``
+    * ``2g.24gb``
+    * ``3g.48gb``
+
+.. _v24.6.0-fixed-issues:
+
+Fixed Issues
```

Comment:
Let's add bullets for the following:

https://github.com/NVIDIA/gpu-operator/issues/782
- This was a bug with NFD. If the NFD worker pod was deleted / restarted on a node, all NFD node labels were removed from the node, therefore leading to all GPU Operator operands to get restarted. 

https://github.com/NVIDIA/gpu-operator/issues/761
- This bug was described here: https://github.com/NVIDIA/cloud-native-docs/pull/76

Reply:
Here is a possible improvement we can call out: https://github.com/NVIDIA/gpu-operator/issues/802